x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/crawler_logic.ts

```diff
@@ -45,7 +45,7 @@ interface CrawlerActions {
 }
 
 export const CrawlerLogic = kea<MakeLogicType<CrawlerValues, CrawlerActions>>({
-  path: ['enterprise_search', 'app_search', 'crawler', 'crawler_overview'],
+  path: ['enterprise_search', 'app_search', 'crawler_logic'],
```

Comment:
_This_ is the culprit for the Overview page not working

Reply:
What is the path used for; I don't recall seeing it in the standalone UI frontend.

Reply:
It's not user facing, it's internal to the codebase.  It's mostly for debugging purposes, but it also determines where actions and values are stored within kea's own internal cache.  This having a conflict with path of `CrawlerOverviewLogic` caused `CrawlerOverviewLogic` to be unable to load its actions into the cache.